packages/wdio-mocha-framework/tests/adapter.test.ts

```diff
@@ -181,6 +181,13 @@ test('prepareMessage', async () => {
     expect(result.type).toBe('afterTest')
     expect(result.title).toBe('foobar')
     expect(result.file).toBe('/foo/bar.test.js')
+    // @ts-ignore
+    adapter._suiteStartDate = Date.now() - 5000
```

Comment:
to access private fields, do
```suggestion
    adapter['_suiteStartDate'] = Date.now() - 5000
```